pkg/config/property.go

```diff
@@ -13,33 +13,33 @@
 package config
 
 const (
-	serverImageName        = "cherestapis.image.name"
-	defaultServerImageName = "quay.io/che-incubator/che-workspace-crd-rest-apis:7.1.0"
-
-	sidecarPullPolicy        = "sidecar.pull.policy"
-	defaultSidecarPullPolicy = "Always"
-
 	pluginRegistryURL = "plugin.registry.url"
```

Comment:
Within the CR, including `che` seems redundant. I would lean towards `workspace.plugin_registry.url` and `workspace.routing.cluster_suffix`. This would also avoid some of the confusion that may arise in e.g. setting `che.*` properties to configure the operator for terminals.

Reply:
I think by not having to set `che.*` properties we also solve a little bit of https://github.com/eclipse/che/issues/17057 and how we can white label the operator